src/test/v2.0/advanced/fdc3.findInstances.ts

```diff
@@ -4,7 +4,8 @@ import { failOnTimeout, wrapPromise } from "../../../utils";
 import { closeMockAppWindow } from "../fdc3-2_0-utils";
 import { IntentUtilityContext } from "../../../context-types";
 import { MetadataFdc3Api } from "../support/metadata-support-2.0";
-import { ContextType, ControlContextType, Intent, IntentApp, RaiseIntentControl2_0 } from "../support/intent-support-2.0";
+import { ContextType, Intent, IntentApp, RaiseIntentControl2_0 } from "../support/intent-support-2.0";
```

Comment:
Bad merge has created an error here I think
```suggestion
import { ContextType, ControlContextType, Intent, IntentApp, RaiseIntentControl2_0 } from "../support/intent-support-2.0";
```

Reply:
fixed